pipelines/datasets/br_anp_precos_combustiveis/tasks.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""
+Tasks for br_anp_precos_combustiveis
+"""
+
+# ! Importando as bibliotecas
+
+import pandas as pd
+import basedosdados as bd
+import unidecode
+
+from datetime import datetime, timedelta
+from dateutil.relativedelta import relativedelta
+
+from prefect import task
+
+from pipelines.utils.utils import (
+    to_partitions,
+)
+from pipelines.datasets.br_anp_precos_combustiveis.utils import (
+    download_files,
+)
+from pipelines.constants import constants
+
+
+@task(
+    max_retries=constants.TASK_MAX_RETRIES.value,
+    retry_delay=timedelta(seconds=constants.TASK_RETRY_DELAY.value),
+)
+def tratamento():
+
+    rename = {
+        "Revenda": "nome_estabelecimento",
+        "CNPJ da Revenda": "cnpj_revenda",
+        "Bairro": "bairro_revenda",
+        "Cep": "cep_revenda",
+        "Produto": "produto",
+        "Valor de Venda": "preco_venda",
+        "Valor de Compra": "preco_compra",
+        "Unidade de Medida": "unidade_medida",
+        "Bandeira": "bandeira_revenda",
+        "Estado - Sigla": "sigla_uf",
+        "Municipio": "nome",
+        "Data da Coleta": "data_coleta",
+        "Nome da Rua": "nome_rua",
+        "Numero Rua": "numero_rua",
+        "Complemento": "complemento",
+    }
+
+    ordem = [
+        "ano",
+        "sigla_uf",
+        "id_municipio",
+        "bairro_revenda",
+        "cep_revenda",
+        "endereco_revenda",
+        "cnpj_revenda",
+        "nome_estabelecimento",
+        "bandeira_revenda",
+        "data_coleta",
+        "produto",
+        "unidade_medida",
+        "preco_compra",
+        "preco_venda",
+    ]
+
+    # ! Carregando os dados direto do Diretório de municipio da BD
+    # Para carregar o dado direto no pandas
+    id_municipio = bd.read_table(
+        dataset_id="br_bd_diretorios_brasil",
+        table_id="municipio",
+        billing_project_id="basedosdados-dev",
+    )
+
+    # ! Tratamento do id_municipio para mergear com a base
+    id_municipio["nome"] = id_municipio["nome"].str.upper()
+    id_municipio["nome"] = id_municipio["nome"].apply(unidecode.unidecode)
+    id_municipio["nome"] = id_municipio["nome"].replace(
+        "ESPIGAO D'OESTE", "ESPIGAO DO OESTE"
+    )
+    id_municipio["nome"] = id_municipio["nome"].replace(
+        "SANT'ANA DO LIVRAMENTO", "SANTANA DO LIVRAMENTO"
+    )
+    id_municipio = id_municipio[["id_municipio", "nome"]]
+
+    download_files()
+
+    for a in {*range(2004, 2023)}:
+
+        df = pd.read_csv(
+            f"input/ca-{a}-01.csv", sep=";", encoding="ISO-8859-1"
+        )  # ! combustiveis automativos 2004 - Primeiro semestre
+
+        dh = pd.read_csv(
```

Comment:
Evitar o excesso de linhas em branco, pois dificulta a leitura do código. Usar esse espacejamento apenas onde for para separar um bloco de outro.